graviti/client/data.py

```diff
@@ -0,0 +1,139 @@
+#
+
+"""Interfaces about data."""
+
+from requests.models import Response
+
+
+def list_data(  # pylint: disable=unused-argument, too-many-arguments
+    url: str,
+    access_key: str,
+    dataset_id: str,
+    draft_number: int,
+    commit: str,
+    segment_name: str,
+    offset: int,
+    limit: int,
+) -> Response:
+    """Get the response of OpenAPI `GET /v1/datasets{id}/data`.
+
+    Arguments:
+        url: The URL of the graviti gas website.
+        access_key: User's access key.
+        dataset_id: Dataset ID.
+        draft_number: The draft number.
+        commit: The information to locate the specific commit, which can be the commit id,
+            the branch name, or the tag name.
+        segment_name: The name of the segment.
+        offset: The offset of the page.
+        limit: The limit of the page.
+
+    Return:
+        The response of OpenAPI.
+
+    """
+
+
+def list_data_details(  # pylint: disable=unused-argument, too-many-arguments
+    url: str,
+    access_key: str,
+    dataset_id: str,
+    draft_number: int,
+    commit: str,
+    segment_name: str,
+    remote_path: str,
+    offset: int,
+    limit: int,
+    is_internal: bool,
+) -> Response:
+    """Get the response of OpenAPI `GET /v1/datasets{id}/data/details`.
+
+    Arguments:
+        url: The URL of the graviti gas website.
+        access_key: User's access key.
+        dataset_id: Dataset ID.
+        draft_number: The draft number.
+        commit: The information to locate the specific commit, which can be the commit id,
+            the branch name, or the tag name.
+        segment_name: The name of the segment.
+        remote_path: The remote path of the data.
+        offset: The offset of the page.
+        limit: The limit of the page.
+        is_internal: Whether the request is from internal.
+
+    Return:
+        The response of OpenAPI.
+
+    """
+
+
+def list_data_urls(  # pylint: disable=unused-argument, too-many-arguments
+    url: str,
+    access_key: str,
+    dataset_id: str,
+    draft_number: int,
+    commit: str,
+    segment_name: str,
+    remote_path: str,
+    offset: int,
+    limit: int,
+    is_internal: bool,
+) -> Response:
+    """Get the response of OpenAPI `GET /v1/datasets{id}/data/urls`.
+
+    Arguments:
+        url: The URL of the graviti gas website.
+        access_key: User's access key.
+        dataset_id: Dataset ID.
+        draft_number: The draft number.
+        commit: The information to locate the specific commit, which can be the commit id,
+            the branch name, or the tag name.
+        segment_name: The name of the segment.
+        remote_path: The remote path of the data.
+        offset: The offset of the page.
+        limit: The limit of the page.
+        is_internal: Whether the request is from internal.
+
+    Return:
+        The response of OpenAPI.
+
+    """
+
+
+def list_mask_urls(  # pylint: disable=unused-argument, too-many-arguments
```

Comment:
can we get rid of the `too-many-arguments` warning?

Reply:
So we need to put this at the beginning of the file? Or add `*` after the `dataset_id`?